Java_Assignment-2/src/FixedOrderPromotion.java

```diff
@@ -0,0 +1,52 @@
+
+
+enum PromotionEnum
+{
+	MONSOON, DISCOUNT50;
+}
+
+class FixedOrderPromotion implements Promotion{
```

Comment:
Use class accessibility while defining class. This should not be in repetitive manner.